ttnn/cpp/ttnn/operations/moreh/moreh_pybind.hpp

```diff
@@ -0,0 +1,13 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+
+#include "ttnn/operations/moreh/moreh_adam/moreh_adam_pybind.hpp"
+
+namespace ttnn::operations::moreh {
+void py_module(py::module& module) { moreh_adam::bind_moreh_adam_operation(module); }
```

Comment:
please split .hpp and .cpp
Follow ccl/all_gather for a good example.
See how .hpp only includes a lightweight pybind_fwd.hpp

Reply:
Sorry for these, it was not too obvious in `examples`, and that is the only documentation I had to go by.

Reply:
If cll/all_gather is a better style than the existing `example`, could you please update the `example` code? @ayerofieiev-tt